src/main/java/quotify_app/app/AppBuilder.java

```diff
@@ -80,6 +83,24 @@ public AppBuilder addLoginUseCase() {
         return this;
     }
 
+    /**
+     * Adds the Landing View to the application.
+     * @return this builder
+     */
+    public AppBuilder addLandingView() {
+        cardPanel.add(landingFactory.getLandingView(), "landing"); // Add Landing View
+        return this;
+    }
+
+    /**
+     * Adds the Landing Use Case to the application.
+     * @return this builder
+     */
+    public AppBuilder addLandingUseCase() {
+        landingFactory.setUpController(viewManagerModel);
```

Comment:
follow the formatting that the other addusecase functions use, it should work for you here